package.json

```diff
@@ -26,7 +26,6 @@
     "glob": "10.3.10",
     "jsdom": "22.0.0",
     "prettier": "2.8.8",
-    "storybook-addon-rtl": "^0.5.0",
```

Comment:
@nathan-ong @somethiiing  fyi follow up from
https://github.com/Cambly/syntax/pull/289/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R29

Storybook specific dependencies should go inside the `storybook` app so moved it to `apps/storybook/package.json`